scripts/popup.js

```diff
@@ -321,10 +321,9 @@ function show_news(){
         var to_check_url=url.replace(/^https?:\/\//,'');
         var final_url=to_check_url.slice(0,to_check_url.indexOf('/'));
         tabId=tabs[0].id;
-        var list_of_sites=["www.huffingtonpost.in","www.nytimes.com","www.forbes.com","www.washingtonpost.com"];
+        var list_of_sites=["www.huffingtonpost.in","www.nytimes.com","www.forbes.com","www.washingtonpost.com","www.huffingtonpost.com"];
```

Comment:
Can you make it `www.huffingtonpost.` to cover all sites?
E.g. there is www.huffingtonpost.gr and countless more. Think globally :)